### PR TITLE
Trivia

### PR DESCRIPTION
**Pull request by christianjokhaji**
Please review. :-)

### CODE REVIEW

File: src/main/java/ca/unknown/bot/entities/RockPaperScissors.java
Comment (Overpoweredplatypus) [inline src/main/java/ca/unknown/bot/entities/RockPaperScissors.java:1]: I like that you added @Override to the method startGame. This reminded me of the fact that you have an abstract class called Game.java

**Review by Overpoweredplatypus — APPROVED** [review]
You've done a lotta work, Christian! I salute you for making an entire parser for the trivia game. I found no problem while reading your files.

**Review by iDasher — APPROVED** [review]
Overall seems good and the API fetcher seems really helpful.

**Review by xinyue296 — APPROVED** [review]
Great work! Thanks for putting up the template for API fetcher and json parser in gson!

**Pull request merged**